packages/components/src/focusable-iframe/index.js

```diff
@@ -35,7 +29,7 @@ class FocusableIframe extends Component {
 			return;
 		}
 
-		const focusEvent = new FocusEvent( 'focus', { bubbles: true } );
+		const focusEvent = new window.FocusEvent( 'focus', { bubbles: true } );
```

Comment:
We don't technically _need_ the `window` here, but it was failing lint without it.